.github/workflows/e2e-agora.yaml

```
@@ -8,13 +8,12 @@ on:
     types: [opened, synchronize, reopened, labeled]
 
 jobs:
-  run-agora-e2e-tests:
+  check-agora-affected:
     # Run in Sage repo on main branch and on all branches in user-owned forks
     if: ${{ github.ref_name == 'main' || github.actor == github.repository_owner }}
-    environment: ${{ github.event_name == 'pull_request_target' &&
-      github.event.pull_request.head.repo.full_name != github.repository && 'agora-pr' || 'agora' }}
-    timeout-minutes: 60
     runs-on: ubuntu-22.04
+    outputs:
+      agora_affected: ${{ steps.agora_affected.outputs.AFFECTED }}
```

We should use the same naming convention / casing for all step outputs (`...outputs.xxx`). The casing should also be updated where the output is exported and wherever needed.

```suggestion
      agora_affected: ${{ steps.agora_affected.outputs.affected }}
```